numpyro/mcmc.py

```diff
@@ -62,15 +63,19 @@ def init_kernel(init_samples,
         z_flat, unravel_fn = ravel_pytree(z)
         momentum_generator = partial(_sample_momentum, unravel_fn)
 
-        find_reasonable_ss = partial(find_reasonable_step_size,
-                                     potential_fn, kinetic_fn, momentum_generator)
+        wa_kwargs = {}
+        # FIXME: compiling find_reasonable_step_size is so slow
```

Comment:
Could you elaborate on this issue? I thought `find_reasonable_step_size` will be called during `wa_init`, but since that is outside the JIT loop, why is it being compiled?

Reply:
I mean the usage of `lax.while_loop` (which does compiling under the hood) in `find_reasonable_step_size` is so slow.